package.json

```
@@ -66,7 +65,7 @@
   },
   "lint-staged": {
     "*.ts": [
-      "yarn prettier --write",
+      "bun prettier --write .",
```

Pretty sure that's gonna write every file per every committed file

```suggestion
      "bun prettier --write",
```

---

you can also remove `bun` as it would use any default package installer.